inference-engine/thirdparty/clDNN/src/primitive_inst.cpp

```diff
@@ -174,21 +175,30 @@ memory::ptr primitive_inst::allocate_output() {
     allocation_type alloc_type = use_lockable_memory ?
                                  engine.get_lockable_preffered_memory_allocation_type(layout.format.is_image_2d())
                                                      : allocation_type::usm_device;
-
+    GPU_DEBUG_GET_INSTANCE(debug_config);
     if (!_network.is_internal() && (_node.can_be_optimized() || _node.is_type<generic_layer>())) {
+        GPU_DEBUG_IF(debug_config->verbose >= 1) {
+            GPU_DEBUG_COUT << "[" << _node.id() << ": output]" << std::endl;
+        }
         return _network.get_memory_from_pool(layout,
                                              _node.id(),
                                              _node.get_memory_dependencies(),
                                              alloc_type,
                                              false);
     } else if (_network.is_internal() && _node.is_output() && _node.is_type<generic_layer>() &&
                engine.supports_allocation(allocation_type::usm_device)) {
+        GPU_DEBUG_IF(debug_config->verbose >= 1) {
```

Comment:
What do you think about increase of verbose level for such prints to 2?

Reply:
I agree. done.